mne_bids/write.py

```diff
@@ -648,8 +657,8 @@ def _sidecar_json(raw, task, manufacturer, fname, datatype, overwrite=False,
         ('RecordingType', rec_type)]
     ch_info_json_meg = [
         ('DewarPosition', 'n/a'),
-        ('DigitizedLandmarks', False),
-        ('DigitizedHeadPoints', False),
+        ('DigitizedLandmarks', True),
```

Comment:
why is this now always TRUE, instead of checking whether landmarks are there first?

Reply:
Based on our discussion in #705 I figured that NAS, LPA, RPA are landmarks. And as far as I know, recordings these points are essential for MEG, so they should always be present. Or am I missing something here?

Reply:
Mh, now that I think about it, empty room recordings would probably be an exception.

Reply:
So, then we could either check what information is present in the `coordsystem.json` (provided that it is written before the sidecar), or we check raw.info['dig'] whether RPA/LPA/NAS are present. Set to True if anything landmark-ish is found, and false otherwise. wdyt?

Reply:
> or we check raw.info['dig'] whether RPA/LPA/NAS are present. Set to True if anything landmark-ish is found, and false otherwise

that sounds good to me